docs/Operators.md

```diff
@@ -7598,6 +7600,8 @@ Other versions of this operator: <a href="Changelog.md#DequantizeLinear-10">10</
 <dd>(Optional) The axis of the dequantizing dimension of the input tensor. Used for per-axis and blocked quantization. Negative value means counting dimensions from the back. Accepted range is `[-r, r-1]` where `r = rank(input)`.</dd>
 <dt><tt>block_size</tt> : int (default is 0)</dt>
 <dd>(Optional) The size of the quantization block (number of times every scale is replicated). Used only for blocked quantization. The block size is a positive integer. Given `x` shape `(D0, ..., Di, ..., Dn)`, `y_scale` shape `(S0, ... Si, ...Sn)` and `axis=i`, the accepted range is `[ceil(Di/Si), ceil(Di/(Si-1))-1]`</dd>
+<dt><tt>output_dtype</tt> : int</dt>
+<dd>(Optional) The output data type. If not supplied, the output data type is inferred from `x_scale` data type (`T2`)</dd>
```

Comment:
Is there a list of how the int values map to the data type? I don't know what values are supplied to this attribute.

Reply:
This issue https://github.com/onnx/onnx/issues/4962 clearly show the data types but I don't really know how you would know the value without some searching.

would it make sense to list integers for the types? (float = 1, float16 = 10, BFloat16 = 16)

Sorry for the question we almost never have to reference the data type by its integer enumeration. I didn't intuitively see the values for this attribute.

Reply:
The enum map for different dtypes is listed in the proto https://github.com/onnx/onnx/blob/main/onnx/onnx.in.proto#L503 I think this is how the conventions used by the rest of the ONNX standard too (ie referring to dtype by enum value).

As the description says here this attribute accepts the same list of dtypes as T2, which is listed a few lines later.